<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="useStandardIconsCheckBox">
        <property name="text">
         <string>Use standard icons</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>No cover image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CoverWidget" name="noCoverWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Click to choose no cover image</string>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid black</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Behaviour</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="hideOnCloseCheckBox">
        <property name="text">
         <string>Hide on close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hiddenCheckBox">
        <property name="text">
         <string>Start hidden</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>300</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="action_NoCover">
   <property name="text">
    <string>No cover</string>
   </property>
   <property name="toolTip">
    <string>Reset no cover image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoverWidget</class>
   <extends>QWidget</extends>
   <header>coverwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
